modules/gapi/include/opencv2/gapi/gstreaming.hpp

```diff
@@ -371,6 +371,29 @@ class GAPI_EXPORTS_W_SIMPLE GStreamingCompiled
 };
 /** @} */
 
+namespace gapi {
+/**
+ * @brief Ask G-API to set specific queue capacity for streaming execution.
+ *
+ * For streaming execution G-API has a special concurrent bounded queue to
+ * fetch next frame while current is being processed. This compilation argument
+ * specifies the capacity of this queue.
+ */
+struct queue_capacity
```

Comment:
I'd put this structure under the `gapi::streaming` namespace as it only makes sense in that execution mode.

Reply:
Done